app/enc/geo/location.go

```diff
@@ -15,7 +15,8 @@ type Location struct {
 }
 
 type LocationWKB struct {
```

Comment:
WKB seems useless  in our domain. We only have location as form of lon\lat, and our domain is not a geometry. 
No meaning is lost if we remove WKB. and It saves mental effort to ponder what this stands for.